flasher.cpp

```diff
@@ -515,47 +472,7 @@ bool Flasher::Erase()
     bool success = false;
 
     if (serial_port_.isOpen()) {
-        if (SendKey()) {
-            success = SendMessage(kEraseCmd, sizeof(kEraseCmd), kEraseTimeoutInMs);
-        }
-    }
-
-    return success;
-}
-
-bool Flasher::GetBoardKey()
-{
-    bool success;
-    const QByteArray magic_string = "NOT_SECURED_MAGIC_STRING_1234567";
-
-    if (board_id_ != magic_string.toHex()) {
-        board_key_ = json_object_.value(board_id_).toString();
-
-        if (board_key_.size() == kKeySizeString) {
-            success = true;
-        }
-        else {
-            success = false;
-            qInfo() << "Board key error";
-        }
-
-    } else {
-        is_secure_boot_ = false;
-        success = true;
-    }
-
-    return success;
-}
-
-bool Flasher::GetBoardKeyFromServer()
-{
-    bool success = false;
-    QByteArray data_input = QByteArray::fromHex(board_id_.toUtf8());
-    QByteArray data_output;
-
-    if (socket::DataTransfer(data_input, data_output)) {
-        board_key_ = data_output.toHex();
-        success = true;
+        success = SendMessage(kEraseCmd, sizeof(kEraseCmd), kEraseTimeoutInMs);
```

Comment:
In Flasher::Flash method, we can also call Erase method instead of calling SendMessage. Line 341.